### PR TITLE
footer

### PR DESCRIPTION
**Pull request by jacekjursza**

### CODE REVIEW

File: extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php [hunk 1]
```diff
@@ -28,7 +28,13 @@ public function testRenderInfobox( $input, $expectedOutput, $description ) {
 		$actualDOM->loadXML($actualOutput);
 		$expectedDOM->loadXML($expectedOutput);
 
-		$this->assertEquals( $expectedDOM->saveXML(), $actualDOM->saveXML(), $description );
+		// lets sanitize expected and actual HTML to not deal with whitespaces
```
Comment (SebastianMarzjan) [inline extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php]: This looks scary. A link to source explaining why this is needed (in the light of DOMDocument attributes set above) should be added here.

**Comment by SebastianMarzjan** [comment]
Some more context is needed in the replace block of the test.
Other than that, LGTM

File: extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php (hunk 1 above)
Comment (jacekjursza) [inline extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php]: without sanitizing, I had something like this:
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 '<?xml version="1.0"?>
 <aside class="portable-infobox">
-  <footer class="portable-infobox-footer portable-infobox-item-margins portable-infobox-header-background portable-infobox-header-font">Footer value</footer>
+  <footer class="portable-infobox-footer portable-infobox-item-margins portable-infobox-header-background portable-infobox-header-font">
+	Footer value
+</footer>
 </aside>
 '

**Comment by SebastianMarzjan** [comment]
Go for it :+1:

**Pull request merged**